tasks/task_mercury_file_wrangling.wdl

```diff
@@ -67,10 +91,26 @@ task sm_metadata_wrangling { # the sm stands for supermassive
 
       return table, excluded_samples
 
+    # set the data file names
+    read1_column_name = "read1_dehosted"
+    read2_column_name = "read2_dehosted"
+    assembly_fasta_column_name = "assembly_fasta"
+    assembly_mean_coverage_column_name = "assembly_mean_coverage"
+
+    # if want to upload clearlabs-generated data and metrics:
+    if (os.environ["using_clearlabs_data"] == "true"):
+      read1_column_name = "clearlabs_fastq_gz"
+      assembly_fasta_column_name = "clearlabs_fasta"
+      assembly_mean_coverage_column_name = "clearlabs_assembly_coverage"
+
+    # if want to upload reads_dehosted instead:
+    if (os.environ["using_reads_dehosted"] == "true"):
```

Comment:
I like this solution to address this specific issue, though I'm thinking that in the long-term, we'll want to harmonize on the language we use for these single-end workflows, e.g. reads_dehosted in that [ncbi_scrubber output](https://github.com/theiagen/public_health_viral_genomics/blob/main/workflows/wf_ncbi_scrub_se.wdl#L28) should probably be read1_dehosted

Reply:
100%. We can make that change in PHB